pkg/apis/operator/v1alpha1/tektonconfig_lifecycle.go

```diff
@@ -21,107 +21,128 @@ import (
 	"knative.dev/pkg/apis"
 )
 
-var (
-	_ TektonComponentStatus = (*TektonConfigStatus)(nil)
+const (
+	ComponentsReady apis.ConditionType = "ComponentsReady"
+)
 
+var (
 	configCondSet = apis.NewLivingConditionSet(
-		DependenciesInstalled,
-		DeploymentsAvailable,
-		InstallSucceeded,
+		PreReconciler,
+		ComponentsReady,
+		PostReconciler,
```

Comment:
I assume these conditions are displayed on status and i think `PreReconciler` and `PostReconciler` words are more towards technical

May i know the reason behind removing old conditions

Reply:
> PreReconciler and PostReconciler words are more towards technical

I agree with this. It would be great if we could rename it with something that indicates the objective/end result of the action. It doesn't has to be precise. Even `PreInstall` `PostInstall` will be better.

Reply:
yeah `PreInstall` `PostInstall` will also be nice
also we do want to break down the component status in some way currently the components cond will show any error
in further interation we do want to find a better way for it

Reply:
Note that if we change what is written in the Status (the name of the condition, …), we need to validate other tools that are depending on this won't be broken (e.g. openshift console, …)

Reply:
(because we are changing the API)

Reply:
yeah in this case we are chaning the api
we have removed the manifest field from status
and updated conditions
what tools do we have to validate? 🤔